index.html

```diff
@@ -11,229 +11,236 @@
+      <div class="desktop">
+        <ul>
+          <li>
+            <a href="">Portfolio</a>
+          </li>
+          <li>
+            <a href="">About</a>
+          </li>
+          <li>
+            <a href="">Contact</a>
+          </li>
+        </ul>
+      </div>
+      <div class="mobile">
+        <img src="images/menu.svg" alt="">
+      </div>
     </nav>
   </header>
-  <section>
+  <section class="home-media">
     <div class="headline">
-      <span class="background-image">
-        <!-- <img src="images/header.svg" alt=""> -->
-      </span>
-      <h1 class="lines"> Hey there <br>
-      I'm Esther <br>
-        <span class="middle">I'm a Software Developer</span>
+      <h1 class="mobile-headline">
+        Hey there. <br >
+        I'm Esther. <br>
+        <span>
+          I'm a software developer
+        </span>
+      </h1>
+      <h1 class="desktop-headline">
+        Hey there.
+        I'm Esther. <br>
+        <span>
+          I'm a software developer
+        </span>
       </h1>
-      <p class="description"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
-      don’t hesitate to contact me.</p>
+    </div>
+    <p class="description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+    <div class="social-icons">
       <ul class="socials">
         <li><img src="images/github.svg" alt="github"></li>
         <li><img src="images/linkdin.svg" alt="linkeldn"></li>
         <li><img src="images/twitter.svg" alt="twitter"></li>
         <li><img src="images/angellist.svg" alt="angellist"></li>
         <li><img src="images/medium.svg" alt="medium"></li>
       </ul>
-      <div>
-      </div>
+    </div>
+    <div class="down-arrow">
+      <img src="./images/Disabled.svg" alt="">
     </div>
     <div class="button">
       <img src="images/vector.svg" alt="normalbottun">
     </div>
   </section>
-  <div class="line-break">
-    <h2 class="works">My Recent <br> Works</h2>
-  </div>
   <section class="section-placeholder">
-    <div  class="placeholder-container">
-      <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
-      <div class="text-container">
-        <h5  class="header2">Multi-Post Stories <br> Gain+Glory</h5>
-        <ul class="text">
-          <li class="text-ruby">Ruby on Rails</li>
-          <li class="text-css">CSS</li>
-          <li class="text-java">JavaScript</li>
-          <li class="text-html">HTML</li>
-        </ul>
-        <div class="button1"><button class="button-text" type="button">See project </button></div>
+    <h2 class="works desktop-head">My Recent Works</h2>
+    <h2 class="works mobile-head">My Recent <br> Works</h2>
+    <span class="line-break"></span>
+    <div class="project-cards">
```

Comment:
Well done! As per the previous review, you made significant improvements to your cards. However, there are still some adjustments needed:

- [ ] For the mobile design, the card section should have 24px padding on the left and right sides. Currently, it is missing in your code. Additionally, I noticed that you have fixed dimensions for the cards, which can cause a vertical scrollbar to appear when padding is added. To solve this, you can use approaches like combining min-width and max-width or using responsive values in your CSS rules, such as percentages (e.g., width: 80%). Please consider updating your code. 🙏
- [ ] The gap between the cards should be 24px as per the design, but currently, it is set to 20px. Please adjust it accordingly. 🙏
- [ ] Each tag should have 12px padding on the left and right sides, but you have it set to 18px. Please update it to match the design requirements.
- [ ] There should be a gap after the button, but currently, there is none. Please add an appropriate gap according to the design.


![Frame 37](https://github.com/essiewanjiru99/essie-portfolio/assets/55840999/f0cde79c-28d3-4280-abcb-879d9bb00edc)

Thank you for your hard work! 🙏